nvflare/lighter/cc_provision/impl/cc.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import Any, Dict, Optional, Type
+
+from nvflare.app_opt.confidential_computing.cc_manager import SHUTDOWN_JOB, SHUTDOWN_SYSTEM
```

Comment:
This package name is really long, can we just use cc for confidential_computing?

Reply:
this needs to be done later, as this package is already there for a while